src/uss/USSTree.ts

```diff
@@ -0,0 +1,529 @@
+/*
```

Comment:
This file is probably a leftover from when you merged in the monorepo changes, and should be removed.